nemo/collections/tts/torch/data.py

```diff
@@ -593,6 +622,12 @@ def general_collate_fn(self, batch):
                 ] = align_prior_matrix
             if Pitch in self.sup_data_types_set:
                 pitches.append(general_padding(pitch, pitch_length.item(), max_pitches_len))
+
+            if Pitch in self.sup_data_types_set:
+                voiced_masks.append(general_padding(voiced_mask, pitch_length.item(), max_pitches_len))
+            if Pitch in self.sup_data_types_set:
+                p_voiceds.append(general_padding(voiced_mask, pitch_length.item(), max_pitches_len))
```

Comment:
Let's use the actual data type and not `Pitch`

Reply:
Done